CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+Changes:
+- Require puma 6.0 or newer
```

Comment:
@harmjanblok Judging by [this comment](https://github.com/harmjanblok/puma-metrics/pull/43#issuecomment-1279130206), I assume that's how you want to proceed, right?